files/build_templates/sonic_debian_extension.j2

```diff
@@ -668,8 +673,10 @@ sudo LANG=C DOCKER_HOST="$DOCKER_HOST" chroot $FILESYSTEM_ROOT docker info
 {% set imagefilepath = image.split(':')|first -%}
 {% set imagefilename = imagefilepath.split('/')|last -%}
 {% set imagename = imagefilename.split('.')|first -%}
+function {{imagename}}()
```

Comment:
Why dynamic function name? Can we use static function name and pass parameter instead?